lib/view_component/base.rb

```diff
@@ -152,6 +152,14 @@ def request
     mattr_accessor :test_controller
     @@test_controller = "ApplicationController"
 
+    # Option to configure if render monkey patches should be inluded or not.
+    # If set to true and rails version < 6.1 render monkey patches will be included
+    # If set to false render moneky patches will not be included but instead the method
+    # render_component will be included in ActionView and ActionController which can be used
+    # for rendering components.
+    mattr_accessor :render_monkey_patch_enabled, instance_writer: false, default: true
+
+
```

Comment:
```suggestion
```

Reply:
Done in 10bdc944593c8b52f209630571c887df8e8a0bc8